### ISSUE TITLE
Most of post meta changes are not saved

### ISSUE THREAD

**Issue by naokomc** [WordPress/revisions-extended#85]
I tried these as a revision, but none of them were saved. 
The excerpt was correctly updated and I could see that in the "scheduled update" screen and the revision. 

- Change the Post Format
- Check "Stick to the top of the blog"
- Update post slug
- Change categories
- Add a tag
- Add a featured image
- Change discussion settings (uncheck both boxes)

Maybe these changes are not meant to be scheduled? Then that should be clearly indicated – maybe when a user clicks the "Crate update" button.

**Comment by coreymckrill** [comment]
This is actually intentional, only the post title, content, and excerpt are saved in an update currently. This follows the same pattern as Core uses for autosaves and revisions (because updates are really just revisions with a different post status). Some of these post meta values (featured image for example) get saved to the original post as soon as they are set (even before you click the "Update" button at the top), so there's not really a good way to include them in an update/revision. I definitely think it would be useful to make it possible to revision some of these other fields, but that should probably happen upstream in Core itself first.

I agree that our documentation should make clear what gets saved, though. Refs #84

**Comment by StevenDufresne** [comment]
I think, within the current UX, we could confirm what was saved in the confirmation window after an update is created. At the very least expectations will be set early as opposed to it going unnoticed until the changes are not found in production later.

**Comment by coreymckrill** [comment]
Maybe something like this in our custom sidebar?

![revision-save-notice](https://user-images.githubusercontent.com/916023/110536365-3aa23e00-80d6-11eb-9253-10e6299c3047.jpg)

**Comment by JustinSainton** [comment]
@coreymckrill Just circling in here to say A) This is SUCH an improvement on existing revisioning plugins, beautiful UX all around and B) Metadata 😣.

Obviously, metadata revisions are [not](https://wordpress.org/plugins/wp-post-meta-revisions/) a [new problem](https://core.trac.wordpress.org/ticket/20564). 

curious if this is being considered within the scope of the project or if any collaborative efforts might be underway towards that end? That's pretty much the only sticking point in a number of our publisher clients being able to use this instead of the other plugins they're currently using.

**Comment by coreymckrill** [comment]
Hey @JustinSainton! Honestly, I don't think post meta revisions are in-scope for this project. It's relying as heavily as possible on existing Core infrastructure, and adding in a feature like that would be a big departure from that.

Is `WP-Post-Meta-Revisions` still functional? Since we are basically just using normal post revisions under the hood, one thing you could try is combining this with that plugin and see how far you can get.

I'll also say that "Phase 3" of Gutenberg is supposed to be around collaboration improvements, so that might be a time when more work gets done on the whole metadata issue.

**Comment by JustinSainton** [comment]
@coreymckrill Yeah, I think that plugin has been more or less abandoned - but looking at the code, I'm sure it would essentially work okay. I'm not certain whether or not the `_wp_autosave_post_revisioned_meta_fields` method would still work as expected...but with enough finagling, it might!

To your point around Phase 3 - a year from now, this may all be a non-issue. 

Thanks for your input, totally makes sense!
